mmdet/models/detectors/base.py

```diff
@@ -1,13 +1,18 @@
-from abc import ABCMeta, abstractmethod
-from collections import OrderedDict
-
+import cv2
 import mmcv
 import numpy as np
+import pycocotools.mask as maskUtils
 import torch
 import torch.distributed as dist
 import torch.nn as nn
+import warnings
+from abc import ABCMeta, abstractmethod
+from collections import OrderedDict
+from contextlib import contextmanager
+from functools import partial
```

Comment:
Imports order looks weird.

Reply:
As in [ote](https://github.com/openvinotoolkit/mmdetection/blob/ote/mmdet/models/detectors/base.py) branch. What exactly confuses you?

Reply:
The main that I care at this point is diff to upstream -- it's better to make it as small as possible. while keeping all the added features functional. Though, as soon as we have lots of other linter issues with our code, you may leave it as is.